site/en/r2/tutorials/distribute/keras.ipynb

```diff
@@ -1,25 +1,38 @@
 {
```

Comment:
<p>&gt; <code>MirroredStategy</code>&nbsp;is the only distribution strategy currently available in TensorFlow core.</p><p><br></p><p>Maybe remove this? We would have to change it in the future, so it may be a good idea to not include it.</p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/333/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
According to the [Distribution Strategy README](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/contrib/distribute) it also no longer accurate.